feedback.md

```diff
@@ -0,0 +1,47 @@
+INFO 343 Final Project Proposal
+
+Describe what your team is proposing to build
+
+We propose to make a website with quizzes that generate a result based on the user's answers. User will be prompted with a multiple choice quiz embellished with pictures and an intuitive UI. After this, the quiz will spit out a result based on the answers to said question. Pre-rendered example quizzes will be provided for users to play around with at launch. We are also proposing to add a feature to this website which allows the user the tools to build their own Buzzfeed-esque quiz. Users should be able to save quizzes after creation. This, however, is a stretch goal.
+
+Possible Titles:
+
+Boozefeed Quizzes
+Buzfeed (only one ‘z’)
+Buzzkill Quizzes
+Online Quiz Maker
+Example Quizzes:
+
+Which booze are you?
+Which kind of hangover are you?
+We can tell what you’re REALLY like based on your Chipotle order!
+Which mid-life crisis Kanye West outfit are you?
+On a scale of 1-10, how “Pizza Rat” are you?
+Reach Goals:
+
+Saving user results
```

Comment:
Does not feel like a reach - totally attainable. Graphing statistics would be a reach/strech goal.